vignettes/a07_updating_rsyncrosim_scripts_to_v2.Rmd

```diff
@@ -37,78 +37,107 @@ Note that when you remove a package from your library any associated datasheets
 
 ## Deprecated functions
 
-In SyncroSim 3, the concept of addon packages no longer exists. All addon packages have either been converted to standalone packages (e.g., `burnP3PlusPrometheus`) and can be added to a library without having to load a base package first, or incorporated directly into the base packages (e.g., `stsimsf` is now part of `stsim`). Therefore, the following functions have been removed:
+In SyncroSim 3, the concept of addon packages no longer exists. All addon packages have either been converted to standalone packages (e.g., `burnP3PlusPrometheus`), and can be added to a library without having to load a base package first, or incorporated directly into the base packages (e.g., `stsimsf` is now part of `stsim`). Therefore, the following functions have been removed:
 
 ```{r deprecated, eval=FALSE}
 addon()
 disableAddon()
 enableAddon()
+
+datasheetRaster() # previously deprecated function. Use datasheetSpatRaster() instead.
```

Comment:
This seems a bit unrelated to the above text to include these other functions in the same code chunk. I think each group of deprecated functions deserves a little bit of text explaining why they are no longer needed.